File: app/src/main/java/com/example/retrofit_pokemon/retrofit2/Urls.java
package com.example.retrofit_pokemon.retrofit2;

import com.google.gson.JsonArray;
import com.google.gson.JsonObject;

import retrofit.Callback;
import retrofit.http.GET;
import retrofit.http.Path;

public interface Urls {


    @GET("/api/information")
    void wsEmergencias(Callback<String> cb);

    /*@GET("/api/news/{uno}")
    void wsNoticias(@Path("uno") String uno, Callback<String> cb);*/
    @GET("/api/news")
    void wsNoticias(Callback<String> cb);

    @GET("/api/covid")
    void wsCovid(Callback<String> cb);

    @GET("/api/centres")
    void wsCentros(Callback<String> cb);

}

File: app/src/main/java/com/example/retrofit_pokemon/retrofit2/Urls.kt
package com.example.retrofit_pokemon.retrofit2

import retrofit.Callback
import retrofit.http.GET

interface Urls {
    @GET("/api/information")
    fun wsEmergencias(cb: Callback<String?>?)

    /*@GET("/api/news/{uno}")
    void wsNoticias(@Path("uno") String uno, Callback<String> cb);*/
    @GET("/api/news")
    fun wsNoticias(cb: Callback<String?>?)

    @GET("/api/covid")
    fun wsCovid(cb: Callback<String?>?)

    @GET("/api/centres")
    fun wsCentros(cb: Callback<String?>?)
}